systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/KafkaRollerST.java

```diff
@@ -322,6 +328,107 @@ void testKafkaPodPendingDueToRack(ExtensionContext extensionContext) {
         KafkaUtils.waitForKafkaDeletion(namespaceName, clusterName);
     }
 
+    /**
+     * @description This test case verifies the rolling update behavior of Kafka controller nodes under specific conditions.
+     * It focuses on ensuring that changes in Kafka configuration affect only the intended node pools, particularly
+     * the controller nodes, while leaving others like broker nodes unaffected.
+     *
+     * @steps
+     *  1. - Assume that KRaft mode is enabled and the installation method is bundle only.
+     *  2. - Initialize the test environment and create a Kafka instance with a single replica.
```

Comment:
Just a nit. But I think this comment is confusing. Given you use node pools and KRaft, the replicas in the Kafka CR is ignored. So You do not really create a Kafka instance with single replica.

Reply:
Yeah, I see okay removing.